benchmark/async_hooks/promises.js

```diff
@@ -0,0 +1,39 @@
+'use strict';
+const common = require('../common.js');
+
+const bench = common.createBenchmark(main, {
+  n: [1e6],
+  method: [
```

Comment:
Nit: These aren't methods. They're descriptive strings, right? I don't have a better idea than `method` though, so maybe it's fine and if someone comes up with something better in the future, they can PR that in.

Reply:
What about:

```
tracking: [
  'enabled',
  'disabled'
]
```

Reply:
(If we do make this change, which I like, we'll need to add `tracking` to `test-benchmark-async-hooks.js`.)